### PR TITLE
Fix/file selector

### PR DESCRIPTION
**Pull request by otigreros**
change the color for the file selector when its disable

### CODE REVIEW

File: app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt [hunk 1]
```diff
@@ -205,4 +205,33 @@ class FileSelectorField @JvmOverloads constructor(
...
+    }
+
+    fun setEnable(isEnableParam: Boolean) {
+
+        clContent?.isEnabled = isEnableParam
+        ivIcon?.isEnabled = isEnableParam
+        tvTitle?.isEnabled = isEnableParam
+        tvTapAction?.isEnabled = isEnableParam
+
+        ivIcon?.setColorFilter(context.resources.getColor(R.color.emerald_disable))
```
Comment (ogallon) [inline app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt]: ??

File: app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt [hunk 2]
```diff
@@ -59,90 +55,81 @@ class FileSelectorField @JvmOverloads constructor(
...
         clContent?.setOnClickListener(this)
-        setOnClickListener(this)
 
         addView(view)
     }
 
     override fun onClick(v: View?) {
-        showFileSelectorDialog()
+        fileSelectorOptionsList?.let {
+            if (it.size > ONE) {
```
Comment (diriarte-condor) [inline app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt]: change . to a meaningful constant

File: test/src/main/res/layout/activity_edittextdatefield_disable_enable_test.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:orientation="vertical"
+              android:padding="16dp"
```
Comment (diriarte-condor) [inline test/src/main/res/layout/activity_edittextdatefield_disable_enable_test.xml]: resources

Comment (otigreros) [reply test/src/main/res/layout/activity_edittextdatefield_disable_enable_test.xml]: its a test view

File: app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt (hunk 2 above)
Comment (otigreros) [reply app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt]: its a file from another file

**Pull request merged**